pkg/identity/identitysync/okta.go

```diff
@@ -133,28 +136,36 @@ func (o *OktaSync) ListUsers(ctx context.Context) ([]identity.IDPUser, error) {
 
 	log.Debugw("listing all okta users")
```

Comment:
This log is in the original code but weird to have it here. Can we move it after the pagination?

Reply:
Why? Looks like it indicates that listing process has been started which makes sense to keep it here.